### PR TITLE
GraphQL - Nested Qualified Properties in Selection Set

### PR DESCRIPTION
**Pull request by abhishoya-gs**
#### What type of PR is this?
Improvement

#### What does this PR do / why is it needed ?
This PR adds support for passing arguments in nested fields in a GraphQL query. In pure this can be represented by qualified property. 

#### Which issue(s) this PR fixes:

Fixes #

#### Other notes for reviewers:
The issue is of unique variable naming in graph fetch tree, since we only have global variables to a lambda function. Hence, we have some logic to handle this by modifying variable names by prepending them with path.

### CODE REVIEW

File: legend-engine-xt-graphQL-query/src/test/java/org/finos/legend/engine/query/graphQL/api/test/TestGraphQLAPI.java [hunk 1]
```diff
@@ -211,6 +245,43 @@ public void testGraphQLExecuteDevAPI_ProcessingTemporalMilestoning_Root() throws
         Assert.assertEquals(expected, responseAsString(response));
     }
 
+    @Test
+    @Ignore // TODO - Fails as date is not supported as an argument now
```
Comment (epsstan) [inline legend-engine-xt-graphQL-query/src/test/java/org/finos/legend/engine/query/graphQL/api/test/TestGraphQLAPI.java:249]: Please move this test into a separate PR

**Review by epsstan — CHANGES_REQUESTED** [review]

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Pull request closed without merging**